R/qenv-join.R

```diff
@@ -9,6 +9,11 @@
 #' - more cases to be done
 #' @param x (`qenv`)
 #' @param y (`qenv`)
+#' @param force (`logical(1)`) whether to turn off automatic validation and duplicated code removal, by default `FALSE`.
```

Comment:
What do you think about explaining objects in `y` overwrite those in `x`?

Reply:
agreed